src/index.html

```diff
@@ -9,6 +9,7 @@
   <link rel="preconnect" href="https://fonts.gstatic.com">
   <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@300;400;500&display=swap" rel="stylesheet">
   <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <script src="https://maps.googleapis.com/maps/api/js?key=AIzaSyCIXdo9Te7uZv_DwombQG4varVXRlyNsmo&libraries=places"></script>
```

Comment:
This key also

Reply:
This key actually isn't doing anything, but I make another commit to leave it out until I'm sure it's safe and I actually start working on the story that needs it

Reply:
also google api keys can be restricted and front end keys really cannot be hidden, so it's all good.